stays/management/commands/recoveryimport.py

```diff
@@ -0,0 +1,77 @@
+from django.core.management.base import BaseCommand, CommandError
+import csv
+from django.db import transaction
+from customers.models import Person, Provider, Pet
+from stays.models import Stay, Review
+import os
+
+def get_or_create_pets(owner, pet_names):
+    pets_list = []
+    for name in pet_names.split("|"):
+        pet, _ = Pet.objects.get_or_create(person=owner, name=name)
+        pets_list.append(pet)
+    return pets_list
+
+def get_or_create_stays(owner, provider, start_date, end_date, pets):
+    stay, _ = Stay.objects.get_or_create(owner=owner, provider=provider, start_date=start_date, end_date=end_date)
+    stay.pets.add(*pets)
+    return stay
+
+class Command(BaseCommand):
+    help = 'New Person, Provider, Pet, Stay and Review are added to database'
+  
+    def add_arguments(self, parser):
+        parser.add_argument('csv_file', nargs='+', type=str)
+
+    def handle(self, *args, **options):
+        imported_rows = 0
+        rows_with_errors = 0
+        if not os.path.isfile(options['csv_file'][0]):
+            raise CommandError(self.style.ERROR("Please enter a valid path"))
+
+        for csv_file in options['csv_file']:
+            dataReader = csv.DictReader(open(csv_file))
+            for row in dataReader:
+                try:
+                    self.get_data(row)
+                    imported_rows += 1
+                except Exception as e:
+                    rows_with_errors += 1
+                    self.stdout.write(self.style.ERROR(e))
+                    continue
```

Comment:
the continue here is not necessary as you are at the end of the for. If you want to continue from the outter for, assign a variable and check it outside this for